Added functionality to add a user to the Firebase list

dpberry552 opened the pull request:
The "Add New User" button will now pass the text from the new user input
box to the "addUser" method, which calls a service to add the new name
to the list of usernames stored in Firebase.

dpberry552 committed:
Added functionality to add a user to the Firebase list

The "Add New User" button will now pass the text from the new user input
box to the "addUser" method, which calls a service to add the new name
to the list of usernames stored in Firebase.

box86rowh commented on src/app/twitch-manage-users/twitch-manage-users.component.html:
```diff
@@ -19,15 +12,24 @@
         <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-        <button type="button" class="btn btn-primary">Save changes</button>
+        <button type="button" class="btn btn-danger">Delete Selected</button>
+        <button (click)="addNewUser(newUser)" type="button" class="btn btn-primary">Add New User</button>
```
you can just do - addNewUser() here

box86rowh commented on src/app/twitch-manage-users/twitch-manage-users.component.ts:
```diff
@@ -5,16 +5,25 @@ import { AngularFire, FirebaseListObservable } from 'angularfire2';
     console.log(this.userNames);
   }
 
+  addNewUser(un: string){
```
you can just do addNewUser() - no need to pass the username in, it is already in scope, you can just refer to this.userName within this function

box86rowh reviewed (changes requested):
see comments for requested changes

dpberry552 committed:
Refactored addNewUser method

No longer unnecessarily passing the username as a parameter to the
addNewUser method

dpberry552 commented:
Made the change...did you see it come through?

box86rowh commented:
yes! Merging...
